apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -639,11 +635,11 @@ Keep in mind you are not just generating an example; you are generating an actua
   );
 
   const files: Record<string, string> = {};
-  if (results?.html) {
+  if (results.html && html !== undefined) {
```

Comment:
I'm somewhat confused by this logic. Why are we checking both `results.html` and `html`? I would've expected us to check and serialize either one or the other.

Reply:
This was fixed in another PR.